bindings/python/setup.py

```diff
@@ -0,0 +1,67 @@
+"setup.py"
```

Comment:
add a `#!/usr/bin/python3` and flag this as executable, it may help if people want to manually execute this